playerctl/playerctl-daemon.c

```diff
@@ -108,6 +110,11 @@ static void player_update_properties(struct Player *player, const char *interfac
         }
         GVariant *cache_value = g_variant_dict_lookup_value(&cached_properties, key, NULL);
         if (cache_value != NULL) {
+            if (is_player_interface && g_strcmp0(key, "PlaybackStatus") == 0) {
```

Comment:
You've missed the case here where PlaybackStatus is not included in the changed properties given by the signal which is a possible case.

Reply:
This is actually important. This would break a whole bunch of tests in test_daemon.py that assume changing any metadata property will grab focus of the active player. (We could rewrite those tests entirely if we wanted to impose the new behavior). I am just assuming that we will only block those players (i.e. spotify) they send the PlaybackStatus property even when it hasn't changed.

Reply:
> This would break a whole bunch of tests in test_daemon.py that assume changing any metadata property will grab focus of the active player.

This means your changes are not covered by the test suite.